text/0000-edition-based-method-disambiguation.md

```diff
@@ -0,0 +1,202 @@
+- Feature Name: `edition_based_method_disambiguation`
+- Start Date: 2022-03-04
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+This RFC proposes a way to introduce new trait methods that conflict with
+existing trait methods in downstream crates in a backwards compatible fashion.
+We do so by annotating new methods with the edition they're introduced in. Then
+when ambigutity is detected between a new method in the standard library and an
+existing downstream method the compiler will check if the crate edition matches
+the edition that the method was introduced in. If it does we pick the
+presumably pre-existing downstream method and output a warning that there was
```

Comment:
I think "presumably pre-existing downstream" is a key phrase to reconsider, because the compiler has no idea about the order of existence. Maybe we want to say something like "pre-edition" instead? (in most uses of "existing" throughout)

It seems to me that it would even be possible for a downstream crate to add new ambiguities any time later and take precedence, as long as they stay on an older edition.